tests/test_pod_exec.py

```diff
@@ -1,24 +1,39 @@
+"""
+This module contains the function to run command on any pod.
+By default it runs on rook-ceph-tool box pod 
+and the default the command is "ceph osd df"
+"""
+
 import os
 from kubernetes import client, config
 os.sys.path.append(os.path.dirname(os.getcwd()))
-
 from ocs import pod
 
+def run_command_on_pod(cmd, label_selector='app=rook-ceph-tools'):
+    """
+    This function provides the ability to execute commands on any pod,
+    by default the command runs on tool-box pod
+
+    Args:
+        cmd (str): The command to execute on the pod
+        label_selector (str): The label of the pod
+                        By default the label will be "app=rook-ceph-tools"
+
+    Returns:
+        ret (int): The return value
+        out (str): If output exists
```

Comment:
The function returns an object called 'V1PodList':

kubernetes.client.models.v1_pod_list.V1PodList